src/helpers/label.ts

```diff
@@ -14,13 +17,13 @@ export const listLabelsForRepo = async (per_page?: number, page?: number): Promi
   });
 
   if (res.status === 200) {
-    return res.data.map((i) => i.name);
+    return res.data;
```

Comment:
Why

Reply:
I guess someone wanted to use all the info of labels